docker-compose.yaml

```diff
@@ -11,9 +13,64 @@ services:
       - postgres_data:/var/lib/postgresql/data
     ports:
       - "5432:5432"
-# next
-# https://hub.docker.com/r/prom/prometheus/
-# https://hub.docker.com/r/prometheuscommunity/postgres-exporter (probably)
+    networks:
+      - monitoring
+
+  postgres_exporter:
+    image: wrouesnel/postgres_exporter:latest
+    container_name: postgres_exporter
+    restart: always
+    environment:
+      DATA_SOURCE_NAME: postgresql://user:password123!@postgres:5432/klikInc?sslmode=disable
+    ports:
+      - "9187:9187"
+    networks:
+      - monitoring
+    depends_on:
+      - postgres
+
+  node_exporter:
+    image: prom/node-exporter:latest
+    container_name: node_exporter
+    restart: always
+    ports:
+      - "9100:9100"
+    networks:
+      - monitoring
+
+  prometheus:
+    image: prom/prometheus:latest
+    container_name: prometheus
+    restart: always
+    ports:
+      - "9090:9090"
+    volumes:
+      - ./prometheus.yml:/etc/prometheus/prometheus.yml
+    networks:
+      - monitoring
+    command:
+      - "--config.file=/etc/prometheus/prometheus.yml"
+    depends_on:
+      - postgres_exporter
+      - node_exporter
+
+  grafana:
+    image: grafana/grafana:latest
+    container_name: grafana
+    restart: always
+    environment:
+      - GF_SECURITY_ADMIN_USER=admin
+      - GF_SECURITY_ADMIN_PASSWORD=admin
+    ports:
+      - "3000:3000"
+    networks:
+      - monitoring
+    depends_on:
+      - prometheus
+
+networks:
```

Comment:
You can delete networks use everywhere, because `bridge` is automatically used so it's redundant